commit 59b70c5e97d4d6a59092edcef914e354f2940aa8
Author: Michael Still <mikal@stillhq.com>
Date:   Thu Apr 23 18:14:34 2026 +1000

    Setuptools released v82.0.0 which does not have pkg_resources.
    
    Pin setuptools in test-requirements.txt as well. The requirements.txt
    pin is already present upstream.
    
    Change-Id: I55851cb5d8f0bb089bd9f49e70bb18a5f270573a
    Signed-off-by: Michael Still <mikal@stillhq.com>

diff --git a/test-requirements.txt b/test-requirements.txt
index 7e3178bd6..da07ce638 100644
--- a/test-requirements.txt
+++ b/test-requirements.txt
@@ -11,3 +11,4 @@ hacking>=7.0.0,<=7.1.0 # Apache-2.0
 oslotest>=3.2.0 # Apache-2.0
 stestr>=2.2.0 # Apache-2.0
 testtools>=2.2.0 # MIT
+setuptools>=64.0.0,<82.0.0 # PSF/ZPL
